File: KhamBenh/BenhNhanObject/BenhNhan.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace BenhNhanObject
{
    [Serializable]
    public class BenhNhan
    {
        public string benhNhanID { get; set; }
        public string soCMND { get; set; }
        public string hoTen { get; set; }
        public string diaChi { get; set; }
        public BenhNhan(){}
        public BenhNhan(string id, string soCMND, string fname, string diaChi)
        {
            this.benhNhanID = id; this.hoTen = fname; this.soCMND = soCMND; this.diaChi = diaChi;
        }
        public override string ToString()
        {
            return benhNhanID + "\t" + soCMND + "\t" + hoTen + "\t" + diaChi;
        }

    }
}

File: GuiKhamBenh/src/data/BenhNhan.java
package data;
import java.io.Serializable;
import java.util.Date;
import javax.xml.bind.annotation.XmlAccessType;
import javax.xml.bind.annotation.XmlAccessorType;
import javax.xml.bind.annotation.XmlRootElement;
import javax.xml.bind.annotation.XmlType;


@XmlType(name="",propOrder={"benhNhanID","soCMND","hoTen","diaChi"})
@XmlRootElement(name ="BenhNhan")
@XmlAccessorType(XmlAccessType.PROPERTY)
public class BenhNhan implements Serializable{
	/**
	 * 
	 */
	private static final long serialVersionUID = 1L;
	private String benhNhanID;
	private String soCMND;
	private String hoTen;
	private String diaChi;
	public String getBenhNhanID() {
		return benhNhanID;
	}
	public void setBenhNhanID(String benhNhanID) {
		this.benhNhanID = benhNhanID;
	}
	public String getSoCMND() {
		return soCMND;
	}
	public void setSoCMND(String soCMND) {
		this.soCMND = soCMND;
	}
	public String getHoTen() {
		return hoTen;
	}
	public void setHoTen(String hoTen) {
		this.hoTen = hoTen;
	}
	public String getDiaChi() {
		return diaChi;
	}
	public void setDiaChi(String diaChi) {
		this.diaChi = diaChi;
	}
	public BenhNhan(String benhNhanID, String soCMND, String hoTen, String diaChi) {
		super();
		this.benhNhanID = benhNhanID;
		this.soCMND = soCMND;
		this.hoTen = hoTen;
		this.diaChi = diaChi;
	}
	public BenhNhan() {
		super();
		// TODO Auto-generated constructor stub
	}
	@Override
	public String toString() {
		return "BenhNhan [benhNhanID=" + benhNhanID + ", soCMND=" + soCMND + ", hoTen=" + hoTen + ", diaChi=" + diaChi
				+ "]";
	}
	
	
}
